src/yunohost/tools.py

```diff
@@ -357,37 +356,27 @@ def tools_update(ignore_apps=False, ignore_packages=False):
                 'fullname': pkg.fullname,
                 'changelog': pkg.get_changelog()
             })
+        logger.info(m18n.n('done'))
 
+    # "apps" will list upgradable packages
     apps = []
     if not ignore_apps:
         try:
             app_fetchlist()
         except MoulinetteError:
+            # FIXME : silent exception !?
             pass
```

Comment:
It has been written 3 years ago
https://github.com/YunoHost/yunohost/commit/5335ffbcbef07e1f815b5f05ca700cc8746c5a4a